giskard_cicd/automation/post_discussion.py

```diff
@@ -19,21 +21,31 @@ def construct_opening(dataset_id, dataset_config, dataset_split, vulnerability_c
     return opening
 
 
-def construct_closing():
+def construct_closing(test_suite_url=None):
+    giskard_hub_wording = f"We've generated test suites according to your scan results! Checkout the [Test Suite in our Giskard Space]({test_suite_url})"
+    
+    if test_suite_url is None:
+        giskard_hub_wording = f"Checkout out the [Giskard Space]({GISKARD_HUB_URL}) and improve your model."
+    
```

Comment:
💬Thought: I think we should emphasize more on the uploaded test suites and lead people to giskard space. I made some changes here, but maybe we can even promote this at the opening if we successfully uploaded the test suites?

Reply:
Sounds good, let's have it before the next tech weekly and collect some feedbacks from the others